src/System.Text.Json/tests/NewtonsoftTests/ILLinkTrim.xml

```diff
@@ -0,0 +1,20 @@
+<linker>
```

Comment:
What's the purpose of adding this? Remove?

cc @steveharter

Reply:
I quess it accidentaly survived rebase, I can delete it.